src/react-app-env.d.ts

```diff
@@ -0,0 +1,64 @@
+/// <reference types="node" />
```

Comment:
You don't need this file at all. Fix tsconfig.json instead.

Reply:
It's from `create-react-app` and I'd like to keep it untouched.